packages/common/src/utils/route.ts

```diff
@@ -293,7 +290,7 @@ export const orderedRoutes = [
   WALLET_AUDIO_PAGE,
   ASSET_DETAIL_PAGE,
   WALLET_PAGE,
-  ALL_COINS_PAGE,
+  COINS_EXPLORE_PAGE,
```

Comment:
i think i used `COINS_EXPLORE_PAGE` for the coins leaderboard..

Reply:
yea this is still that, just moving routes around